src/IMangrove.sol

```diff
@@ -40,16 +39,37 @@ interface IMangrove {
     uint id,
     uint prev
   );
-  event OrderComplete(
+  // tried to add all relevant data for an order.
+  // same
+  event OrderComplete( // this could be deduced from the OfferSuccess events emtitted from "execute" in Mangrove
     address indexed outbound_tkn,
     address indexed inbound_tkn,
     address indexed taker,
+    uint takerWants,
+    uint takerGives,
     uint takerGot,
     uint takerGave,
+    bool fillWants,
+    uint penalty,
+    uint feePaid
```

Comment:
`feePaid` is derivable from last Mangrove's `SetFee` event.

Reply:
From the perspective on self-contained events (which this PR is going for) it makes sense to include the actual fee.

Also, one will need to look at all the taken offers to calculate this precisely due to rounding.